test/unit/full_text_search/searcher_test.rb

```diff
@@ -82,5 +82,41 @@ def test_syntax_error_query
       assert_equal([issue],
                    targets.collect(&:source_record).sort_by(&:id))
     end
+
+    def test_results_ordered_by_descending_registered_time
+      parameters = {
+        order_target: "registered_time",
+        order_type: "desc",
+        search_id: @project.id,
+        news: "1",
+        attachments: "0",
+        limit: "-1"
+      }
+      targets = search(parameters).records
+      searched_news_ids = targets.collect(&:source_id)
+      ordered_news = @project
```

Comment:
`ordered_news_ids` is better for this because this is an array of record IDs not records: 

```suggestion
      ordered_news_ids = @project
```